anitya/__init__.py

```diff
@@ -145,7 +145,8 @@ def log(session, project=None, distro=None, topic=None, message=None):
                              'project: %(project)s',
         'project.edit': '%(agent)s edited the project: %(project)s fields: '
                         '%(changes)s',
-
+        'project.flag': '%(agent)s flagged the project: %(project)s',
```

Comment:
It's your choice. The reason might be kind of long, so I excluded it from the log message, figuring that an admin could check the reason from the Flags view itself. Do you think it'd be good to have it here, as well?